File: lemniscate/src/test/java/com/vlad1m1r/lemniscate/base/models/PointTest.java
package com.vlad1m1r.lemniscate.base.models;

import com.vlad1m1r.lemniscate.TestConstants;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class PointTest {
    @Test
    public void testIfPointsAreTranslated() {
        Point point = new Point(0,30,30,270);
        assertEquals(135, point.x(), TestConstants.DELTA);
        assertEquals(159.5454559326172, point.y(), TestConstants.DELTA);
    }
}

File: lemniscate/src/test/java/com/vlad1m1r/lemniscate/base/models/PointTest.kt
package com.vlad1m1r.lemniscate.base.models

import com.google.common.truth.Truth.assertThat
import org.junit.Test

class PointTest {
    @Test
    fun testIfPointsAreTranslated() {
        val point = Point(0.0f, 30.0f, 30.0f, 270.0f)
        assertThat(point.x).isEqualTo(135.0f)
        assertThat(point.y).isEqualTo(159.545455f)
    }
}